package.json

```diff
@@ -160,8 +160,9 @@
   },
   "files": [
     "build/",
-    "dist/",
-    "flow-typed/",
+    "dist/mapbox-gl*",
+    "dist/style-spec/",
```

Comment:
Restricting the scope of `/dist` to prevent `versions.json`, `versions.jsonp` and any other dev files from being included.